Python/hbayesdm/models/_bandit4arm_2par_lapse.py

```diff
@@ -61,9 +61,9 @@ def bandit4arm_2par_lapse(
         stepsize: float = 1,
         max_treedepth: int = 10,
         **additional_args: Any) -> TaskModel:
-    """4-Armed Bandit Task - 3 Parameter Model, without C (choice perseveration), R (reward sensitivity), and P (punishment sensitivity). But with xi (noise)
+    """N-Armed Bandit Task - 3 Parameter Model, without C (choice perseveration), R (reward sensitivity), and P (punishment sensitivity). But with xi (noise)
 
-    Hierarchical Bayesian Modeling of the 4-Armed Bandit Task 
+    Hierarchical Bayesian Modeling of the N-Armed Bandit Task 
```

Comment:
Seems that you also changed descriptions of existing functions for 4-armed bandit tasks. It would be better to leave as it is, until we reach any agreement regarding it.

Reply:
This comment applies for all the changes in this PR.